libraries/reconstruction/resolution_monotomo.cpp

```diff
@@ -1045,7 +957,7 @@ void ProgMonoTomo::run()
 	outputResolutionImage() = FilteredResolution;
 	outputResolutionImage.write(fnOut);
 
-	if (fnmaskWedge != "")
+	if (fnmaskWedge.isEmpty())
```

Comment:
before you were checking if it's NOT empty

Reply:
Fixed, I put  != "" this is more general, and avoid the use of library functions.